pkgs/tools/networking/curl/default.nix

```diff
@@ -136,6 +131,12 @@ stdenv.mkDerivation rec {
     ++ lib.optionals stdenv.hostPlatform.isWindows [
       "--disable-shared"
       "--enable-static"
+    ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
Wouldn't it be better to use `stdenv.hostPlatform.isDarwin` instead to take care of the cross-compiling cases here?

Reply:
isDarwin is an alias to hostPlatform.isDarwin https://github.com/nix-community/home-manager/blob/master/tests/default.nix#L35-L36